client/src/rpc.rs

```diff
@@ -1631,6 +1632,7 @@ async fn get_server_info(
         pub chain: String,
         pub headers: u32,
         pub blocks: u32,
+        pub headerssynced: Option<bool>,
```

Comment:
Maybe better to use [BlochchainInfo](https://github.com/spacesprotocol/spaces/blob/e268db5706f9c2b86de5f1954911972ed3662a34/client/src/client.rs#L51) type?